src/Tests.Microsoft.Security.Utilities.Core/LocalizationTests.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.Security.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+using System.Globalization;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Tests.Microsoft.Security.Utilities.Core
+{
+    [TestClass]
+    public class LocalizationTests
```

Comment:
Please delete these tests. They will cause us to have to update them any time a string is changed and I don't think it's necessary to unit test the exact string. They'd also fail if we localize and run them in non-English environment.